draft-ietf-dhc-rfc8415bis.xml

```diff
@@ -3129,6 +3127,17 @@ addresses, and this specification is now authoritative for:</t>
           </ul>
 
         </section>
+        <section anchor="RFC8415bis-18.2.13" numbered="true" toc="default">
+          <name>Restarting Server Discovery Process</name>
+          <t>Whenever a client restarts the DHCP server discovery process or
+        selects an alternate server as described in <xref target="RFC3315-17.1.3" />,
+        the client SHOULD stop using any addresses and delegated prefixes for
+        which it has bindings (see <xref target="RFC3315-18.1.6" />)
+        and any previously received other configuration
+        information, and try to obtain new bindings and other configuration
```

Comment:
I'd add extra "if possible". Sometimes it's not possible to stop using a configuration. How do you stop using a hostname or a timezone?

Reply:
Michael agrees with this, as he offered exactly this suggestion [here](https://github.com/dhcwg/rfc8415bis/pull/78#pullrequestreview-2843025846). Resolving.